app/src/main/java/com/hedvig/app/feature/genericauth/GenericAuthViewModel.kt

```diff
@@ -3,14 +3,23 @@ package com.hedvig.app.feature.genericauth
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.hedvig.app.feature.embark.EMAIL_REGEX
+import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.receiveAsFlow
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
 
 class GenericAuthViewModel(
     private val createOtpAttemptUseCase: CreateOtpAttemptUseCase,
 ) : ViewModel() {
+    private val eventChannel = Channel<Event>(Channel.UNLIMITED)
+    val eventsFlow = eventChannel.receiveAsFlow()
```

Comment:
`_events` and `events` is usually what we've been naming those, I find the type suffix to not be super necessary since we get IDE type hints and all that jazz anyway.
I think Oscar thinks the same too, but not too strong opinion on this.

Reply:
Good point :) Will change to the standard we use.

Reply:
Agreed - the style is commonly called hungarian notation, and I believe it to be superfluous when using a modern IDE and a well-behaved statically typed language 😄